### ISSUE TITLE
problems with nycflights13_sql

### ISSUE THREAD

**Issue by pachamaltese** [jonkeane/dbtest#45]
this won't work, returns 0 rows:

```
con_psql <- RPostgreSQL::dbConnect(
  drv = DBI::dbDriver("PostgreSQL"),
  dbname = "dbtest",
  host = "postgres.server",
  user = "dbtest",
  password = "_dbtest_"
)

nycflights13_sql(con_psql)

tbl(con_psql, in_schema("public", "flights")) %>%
  filter(!is.na(tailnum)) %>%
  collect() %>%
  group_by(tailnum) %>%
  count() %>%
  filter(n >= 10L)
```

this will:

```
flights %>%
+   filter(!is.na(tailnum)) %>%
+   collect() %>%
+   group_by(tailnum) %>%
+   count() %>%
+   filter(n >= 10L)

# A tibble: 3,431 x 2
# Groups:   tailnum [3,431]
   tailnum     n
   <chr>   <int>
 1 N0EGMQ    371
 2 N10156    153
 3 N102UW     48
 4 N103US     46
 5 N104UW     47
 6 N10575    289
 7 N105UW     45
 8 N107US     41
 9 N108UW     60
10 N109UW     48
# … with 3,421 more rows
```

**Comment by pachamaltese** [comment]
the problem is that in postgres the writing function sends 1,000 records, I checked connecting with both RPostgres and RPostgreSQL and then trying `select count(*) from public.flights as f` from dbeaver

**Comment by pachamaltese** [comment]
solved in 1d66248
